apps/test-viewer/src/UiProvidersConfig.tsx

```diff
@@ -4,8 +4,8 @@
  *--------------------------------------------------------------------------------------------*/
 import { StagePanelLocation, StagePanelSection, UiItemsProvider } from "@itwin/appui-react";
 import { SelectionMode } from "@itwin/components-react";
-import { IModelConnection } from "@itwin/core-frontend";
-import { EC3Provider } from "@itwin/ec3-widget-react";
+import { IModelApp, IModelConnection } from "@itwin/core-frontend";
```

Comment:
Weird to see `IModelApp` addition when no other changes in this PR reference it. Is it needed? If not - remove.

Reply:
done